exasol/python_extension_common/deployment/language_container_deployer_cli.py

```diff
@@ -80,24 +83,32 @@ def clear_formatters(self):
 
 
 @click.command(name="language-container")
-@click.option('--bucketfs-name', type=str, required=True)
-@click.option('--bucketfs-host', type=str, required=True)
-@click.option('--bucketfs-port', type=int, required=True)
+@click.option('--bucketfs-name', type=str)
+@click.option('--bucketfs-host', type=str)
+@click.option('--bucketfs-port', type=int)
 @click.option('--bucketfs-use-https', type=bool, default=False)
-@click.option('--bucketfs-user', type=str, required=True, default="w")
-@click.option('--bucketfs-password', prompt='bucketFS password', hide_input=True,
-              default=lambda: os.environ.get(BUCKETFS_PASSWORD_ENVIRONMENT_VARIABLE, ""))
-@click.option('--bucket', type=str, required=True)
-@click.option('--path-in-bucket', type=str, required=True, default=None)
+@click.option('--bucketfs-user', type=str)
+@click.option('--bucketfs-password', type=str,
+              default=lambda: os.environ.get(BUCKETFS_PASSWORD_ENVIRONMENT_VARIABLE))
+@click.option('--bucket', type=str)
+@click.option('--saas-url', type=str,
+              default='https://cloud.exasol.com')
+@click.option('--saas-account-id', type=str,
+              default=lambda: os.environ.get(SAAS_ACCOUNT_ID_ENVIRONMENT_VARIABLE))
+@click.option('--saas-database-id', type=str,
+              default=lambda: os.environ.get(SAAS_DATABASE_ID_ENVIRONMENT_VARIABLE))
+@click.option('--saas-token', type=str,
```

Comment:
These values are quite long, entering them manually would be torture.

Reply:
you can copy them into the input prompt. The input prompt is usually used, when you don't want to leak your credentials in your shell history. However, as I said, this can be done later. I think, it simply enough to create a ticket for it.

Reply:
I am not sure I understood. According to the [click doc](https://click.palletsprojects.com/en/8.1.x/options/#dynamic-defaults-for-prompts), you can offer the user a choice, either to accept the default value stored in a config or env. variable, or enter it manually from a terminal. My point is that entering the saas parameters manually is not practical because they are too long. Hence we don't need a prompt here. It may be useful for the db password though.